app/geo/gui/src/ts/twxplore/gui/geo/components/map/Map.tsx

```diff
@@ -24,9 +24,11 @@ import {startQuerying} from "../../actions/map/StartQueryingAction";
 import {finishLoad} from "../../actions/map/FinishLoadAction";
 import {repeatQuery} from "../../actions/map/RepeatQueryAction";
 import {MapFeatureTypeState} from "../../states/map/MapFeatureTypeState";
+import {FilterPanel} from "../filterPanel/FilterPanel";
 //import KeplerGlSchema from "kepler.gl/schemas";
 
 const limit = 500;
+const DEBUG = true;
```

Comment:
It's fine to have this kind of constant, but I'd suggest you use an environment variable. That means you don't have to modify the source to change it.

See Environment.ts in the base lib. You could extend that class with one local to your app, and add a .debug. The values come from the webpack DefinePlugin, which can be keyed off of a command line parameter. Which would then mean you have a start-debug script in package.json.